Arrangement_on_surface_2/doc/Arrangement_on_surface_2/Concepts/ArrangementBottomSideTraits_2.h

```diff
@@ -29,6 +29,7 @@ class ArrangementBottomSideTraits_2 {
 
   /// \name Functor Types
```

Comment:
Are these empty sections of any use?  @MaelRL @sloriot ??

Reply:
In my opinion they are not of any use, I left them as they were already present and just completed them.

Reply:
they are probably left overs.